samples/echoWebhookBot/main.go

```diff
@@ -70,8 +70,7 @@ func main() {
 	// Start the webhook server. We start the server before we set the webhook itself, so that when telegram starts
 	// sending updates, the server is already ready.
 	webhookOpts := ext.WebhookOpts{
-		Listen:      "0.0.0.0", // This example assumes you're in a dev environment running ngrok on 8080.
-		Port:        8080,
+		ListenAddr:  ":http-alt",   // This example assumes you're in a dev environment running ngrok on 8080.
```

Comment:
Do you know how common the `:http-alt` syntax is? I've personally never seen it before, and would probably be more comfortable using `:8080`, if thats OK with you. I've also re-added the "0.0.0.0", to be explicit about what the example listening on; makes it clearer to anyone reading it imo, what do you think? :) 

```suggestion
		ListenAddr:  "0.0.0.0:8080",   // This example assumes you're in a dev environment running ngrok on 8080.
```

Reply:
"0.0.0.0:8080" is good. But we should document that if the host in the address parameter is empty, we listen on all available unicast and anycast IP addresses of the local system (this is what net.Listen does).
Or even better, I think we would default to "localhost:8080" if we assume that we are in a dev env, no need to listen on all interfaces.

Reply:
Sure, that seems fair to me. Localhost should be enough, you're right!